modules/icgc-argo-workflows/prep/metrics/main.nf

```diff
@@ -19,11 +19,11 @@ process PREP_METRICS {
     when:
     task.ext.when == null || task.ext.when
 
-    script: 
-    
+    script:
+    def RNA = (meta.experiment ?: "") == "RNA-Seq"
 
     """
-    main.py \\
+    ${RNA ? 'rnaaln.py' : 'dnaaln.py'} \\
```

Comment:
A case statement might be easier and scalable for different types:
https://github.com/icgc-argo-workflows/argo-modules/blob/19d2393fd88fca055be66cbd375c0147d43d9b7f/modules/icgc-argo-workflows/checkinput/main.nf#L23